src/test/kotlin/no/nav/aap/api/OMDokumenter.kt

```diff
@@ -0,0 +1,28 @@
+package no.nav.aap.api
+
+import org.springframework.core.io.ClassPathResource
+import org.springframework.util.StreamUtils.copyToByteArray
+
+object OMDokumenter {
```

Comment:
Trenger du alle disse metodene? Hva med kun en, som tar inn et filnavn?

Reply:
Er ikke alt for stabil på OM, men forstod på Sturle at det var dette som va strukturen for denne typen klasser. @sturleh har du noen inspill?

Reply:
Fjernet OMDokumenter og henter nå filer fra ArkivConverterTest da dette er det eneste stedet vi bruker filer.